### PR TITLE
TELCODOCS#1255: Added limitations related to creating snapshots and clones in multi-node topology

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s): 4.15
Issue: [TELCODOCS-1255](https://issues.redhat.com/browse/TELCODOCS-1255)

Link to docs preview: https://69298--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by jakobmoellerdev — APPROVED** [review]
LGTM

**Review by jakobmoellerdev — APPROVED** [review]
/lgtm

**Comment by sr1kar99** [comment]
@radeore 
Could you please review this PR?
Thanks.

File: modules/deploying-lvms-on-sno-cluster.adoc
Comment (Srivaralakshmi) [inline modules/deploying-lvms-on-sno-cluster.adoc:53]: @sr1kar99 For example, I tried rewriting this bullet (L53) to keep it simple and in active voice:

If the PVC is not present and available before creating a volume snapshot, then the scheduler in multi-node OpenShift clusters does not provision the volume snapshots on the node where you have deployed the original data source.

I am not sure if it is technically correct. You can check and if all is ok, you can follow this style or a similar style to rewrite.

**Review by Srivaralakshmi — COMMENTED** [review]
@sr1kar99 Excellent work! I have left some suggestions for you to consider. PTAL. Thank you!

File: modules/deploying-lvms-on-sno-cluster.adoc
Comment (sr1kar99) [reply modules/deploying-lvms-on-sno-cluster.adoc]: I have split this point into two separate points:

- Pods relying on the persistent volume claim (PVC) that uses the snapshot data and clone data can be scheduled only on the node where you have deployed the original data source.

- Volume snapshots and volume clones can be created only on the node where you have deployed the original data source.

As I noticed that the action associated with pods is "scheduling", and snapshots and clones is "creating", having a separate point for each will avoid confusion.

File: modules/deploying-lvms-on-sno-cluster.adoc
Comment (sr1kar99) [reply modules/deploying-lvms-on-sno-cluster.adoc]: I have removed the following line:

"The original persistent volume claim (PVC) is in `Bound` state." 

This point is already present in the pre-req for "[Creating volume clone](https://69298--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms#lvms-creating-volume-snapshots-in-single-node-openshift_logical-volume-manager-storage)" and "[Creating volume snapshots](https://69298--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms#lvms-creating-volume-clones-in-single-node-openshift_logical-volume-manager-storage)" procs.

I have added all the limitations in a new module, and added a note with a cross-reference to this module at the beginning of "[Volume snapshots](https://69298--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms#lvms-volume-snapsot-for-sno_logical-volume-manager-storage)" and "[Volume cloning](https://69298--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms#lvms-volume-cloning-for-single-node-openshift-cluster_logical-volume-manager-storage)"

@radeore - Could you please confirm if this update is fine?

File: modules/deploying-lvms-on-sno-cluster.adoc
Comment (sr1kar99) [reply modules/deploying-lvms-on-sno-cluster.adoc]: As confirmed by @radeore, this info also applies to volume clones. Hence, updated the line as follows:

If the PVC is not present and available before creating a volume snapshot or volume clone, then the scheduler in multi-node OpenShift clusters does not provision the volume snapshots and volume clones on the node where you have deployed the original data source.

**Comment by sr1kar99** [comment]
> @sr1kar99 Excellent work! I have left some suggestions for you to consider. PTAL. Thank you!

@Srivaralakshmi Thank you for the thorough review. I have updated the content based on your comments. 
Can you please take a look?
Thank you :)

**Review by jakobmoellerdev — APPROVED** [review]
LGTM

File: modules/lvms-limitations-for-creating-snapshots-and-clones.adoc
Comment (Srivaralakshmi) [inline modules/lvms-limitations-for-creating-snapshots-and-clones.adoc:13]:
```suggestion
* Volume snapshots and volume clones get created only on the node where you have deployed the original data source.
```

To avoid passive voice. Please check with SME for technical accuracy though.

**Review by Srivaralakshmi — COMMENTED** [review]
@sr1kar99 Great job! LGTM. A small nitpick. Please address it if it is a reasonable suggestion and send it more merging. No need for another review round from my side. Thanks!

File: modules/lvms-limitations-for-creating-snapshots-and-clones.adoc
Comment (jakobmoellerdev) [reply modules/lvms-limitations-for-creating-snapshots-and-clones.adoc:13]: Meaning is unaffected by this, change is ok from my side

Comment (sr1kar99) [reply modules/lvms-limitations-for-creating-snapshots-and-clones.adoc:13]: @jakobmoellerdev 
As volume clones and snapshots can be created by users, can we write:

You can create volume snapshots and volume clones only on the node where you have deployed the original data source.

Comment (sr1kar99) [reply modules/lvms-limitations-for-creating-snapshots-and-clones.adoc]: Thank you @jakobmoellerdev, @Srivaralakshmi for the review.
Updated the line.

Comment (sr1kar99) [reply modules/lvms-limitations-for-creating-snapshots-and-clones.adoc]: @Srivaralakshmi -  If we can proceed to merging this PR, could you please add the "peer-review-done" label?
Thanks.

**Review by Srivaralakshmi — COMMENTED** [review]
@sr1kar99 LGTM. Excellent work! Thank you!

As I am new to the peer review squad, I would ask @bmcelvee to give my review the go-ahead. Please wait until Brandi applies the peer-review-done label.

File: modules/lvms-limitations-for-creating-snapshots-and-clones.adoc
Comment (bmcelvee) [inline modules/lvms-limitations-for-creating-snapshots-and-clones.adoc:15]: s/OpenShift/{product-title} 

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#product-title-and-version

**Comment by bmcelvee** [comment]
Just one more edit. Otherwise, LGTM!

File: modules/lvms-limitations-for-creating-snapshots-and-clones.adoc
Comment (Srivaralakshmi) [reply modules/lvms-limitations-for-creating-snapshots-and-clones.adoc:15]: @bmcelvee Good catch! somehow, I missed this one. TY!

Comment (sr1kar99) [reply modules/lvms-limitations-for-creating-snapshots-and-clones.adoc]: @bmcelvee Thank you for the review.
Updated the content based on your comment.

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**
